src/main/java/hudson/plugins/build_timeout/impl/DeadlineTimeOutStrategy.java

```diff
@@ -20,7 +20,7 @@
 /**
  * If the build reaches <tt>deadlineTime</tt>, it will be terminated.
  * 
- * @author Fernando Migu�lez Palomo <fernando.miguelez@gmail.com>
+ * @author Fernando Miguélez Palomo <fernando.miguelez@gmail.com>
```

Comment:
We can't see the difference on the github page, but former line is encoded with Latin1 and the latter line is encoded with UTF-8.